### DISCUSSION TITLE
Build cuCollections on Windows system

### DISCUSSION THREAD

**Discussion by littlesheep12** [NVIDIA/cuCollections#471] [Q&A] [answered]
I followed the instructions in Git Hub and added cuCollection to my project, cmake works fine, but I get an error when I build.
It can not compile and run examples.
If it is convenient, can you tell me about how to configure the corresponding environment **Windows**.

**Comment by PointKernel** [comment, marked as answer]
Can you build on Linux instead? Unfortunately, we don't have a plan to support Windows in the near future (see https://github.com/NVIDIA/cuCollections/issues/195).

**Reply by littlesheep12** [reply]
okay, thx reply.
